led_strip/include/led_strip_types.h

```diff
@@ -22,6 +32,8 @@ typedef struct led_strip_t *led_strip_handle_t;
 typedef struct {
     uint32_t strip_gpio_num; /*!< GPIO number that used by LED strip */
     uint32_t max_leds;       /*!< Maximum LEDs in a single strip */
+    led_pixel_format_t led_pixel_format;
```

Comment:
would you please add some docs beside? We will add API documentation for this library, and the Doxygen tool will warn here (uncommented struct field).

Reply:
It seems I arrived late to the party, master already contains the documentation...